layers/core/filter.py

```diff
@@ -6,7 +6,7 @@
         UNSETVALUE
 
 
-class Filter:
+class Filterv4:
     def __init__(self, domain="mitre-enterprise"):
```

Comment:
If this is the filter object as implemented by layer v4, shouldn't the domain be formatted as `enterprise-attack` instead of `mitre-enterprise`? `mitre-enterprise` is the old domain formatting from versions <=3.0.

Reply:
Yep, missed updating the default.